lib/commands/createcache.js

```diff
@@ -62,7 +74,25 @@ function createCache(opts, request, done){
         "persistent" : true,
         "skipCacheIfElementSizeInKBExceeds" : 512
       }
-
+  
+  if(opts.cacheExpiryByDate){
+    var expirySettings = {
+      "expiryDate" : {
+        "value" : opts.cacheExpiryByDate
+      },
+      "valuesNull" : false
+    }
+    createCachePayload.expirySettings = expirySettings;
```

Comment:
Minor nit, you could do this way easier by using `createCachePayload.expirySettings.expiryDate.value = opts.cacheExpiryByDate`.  All you need to do is override the value already set with the one provided.

Reply:
Code updated to overwrite expiryDate value.